docs/components/attention/vue.md

```diff
@@ -62,13 +86,69 @@ const target = ref(null)
 const showing = ref(false)
 </script>
 <template>
-  <w-box neutral class="h4" ref="target" @mouseenter="showing = true" @mouseleave="showing = false">Hover over me</w-box>
-  <w-attention tooltip bottom :target-el="target?.$el" v-model="showing">
-    <p>Hello Warp!</p>
+  <w-box
+    neutral
+    as="h4"
+    ref="target"
+    @mouseenter="showing = true"
+    @mouseleave="showing = false"
+    @keydown.escape="showing = false"
+    @focus="showing = true"
+    @blur="showing = false"
+    tabindex="0"
+  >
+    <button
+      aria-describedby="tooltip-bubbletext"
+      aria-expanded="true"
+      type="button"
+    >
+      Hover over me
+    </button>
+  </w-box>
+  <w-attention
+    tooltip
+    bottom
+    :target-el="target?.$el"
+    v-model="showing"
+    @focus="showing = true"
+    @blur="showing = false"
+  >
+    <p id="tooltip-bubbletext">Hello Warp!</p>
   </w-attention>
 </template>
 ```
 
+### Accessibility
+
+The attention element handles accessibility automatically by wrapping its slotted content with a `div` that has a default `role`attribute (`role="tooltip"` for tooltip and `role="img"` otherwise), and a default localized `aria-label`.
+
+It is possible to reset the `role` and `aria-label` attributes, and the default `aria-label` can be overridden:
```

Comment:
```suggestion
It is possible to override the `role` and `aria-label` attributes:
```

Reply:
I'm not sure we can write override for the `role` attribute. The user can only reset that attribute but not override it. While they are able to reset + override the `aria-label`